.env.local.example

```diff
@@ -1,14 +1,10 @@
 ADMIN_URL_BASE=admin/
 CLOUDFLARE_API_TOKEN=
 DATABASE_URL=postgres://djangopackages:djangopackages@postgres:5432/djangopackages
-DEBUG=
-DJANGO_DEBUG=
+DJANGO_DEBUG=True
 DJANGO_MAILGUN_API_KEY=
 DJANGO_SENTRY_DSN=
-DJANGO_SETTINGS_MODULE=
-DOMAIN_NAME=
-EMAIL_HOST_USER=
-FRAMEWORK_TITLE=
```

Comment:
FWIW I did need `FRAMEWORK_TITLE` to make it run locally. I got an error without it. (I copied and pasted your new `.env.local.example`)

Reply:
Thank you for this. Maybe it's time for "one settings file to rule them all."